Makefile

```
@@ -43,6 +44,9 @@ install-vm: install-both
 	$(INSTALL_DATA) scripts/webcam.html $(DESTDIR)$(DATADIR)/$(PKGNAME)/scripts
 	$(INSTALL_DATA) scripts/dkms-skip.conf $(DESTDIR)$(SYSCONFDIR)/dkms/v4l2loopback.conf
 	$(INSTALL_DIR) $(DESTDIR)$(DATADIR)/$(PKGNAME)/scripts/v4l2loopback
+	install -d $(DESTDIR)/$(APPLICATIONSDIR)
```

```suggestion
	install -d $(DESTDIR)$(APPLICATIONSDIR)
```
APPLICATIONSDIR already includes leading `/`

Same applies to the two lines below too